### ISSUE TITLE
Why `DB.name` is not exposed?

### ISSUE THREAD

**Issue by sbellem** [wbolster/plyvel#88]
I just wonder why it is not possible to access the name of `DB` instance, e.g.:

```python
>>> from plyvel import DB
>>> db = DB('/tmp/db', create_if_missing=True)
>>> db.name
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'plyvel._plyvel.DB' object has no attribute 'name'
```

I do not really know how `cython` works. I can see from the code that within the `DB` class `self.name` is accessible (e.g. in `__repr__()` and `__str__()`) so I wonder why we cannot access it when using it. Is this a design decision? Just curious.

Thanks!

**Comment by wbolster** [comment]
there's no particular reason. i think by default the attributes of cython classes are visible only at the C level (and not at the python level).

**Comment by sbellem** [comment]
Ok thanks!

I made a tiny PR: #90 that exposes it.

I can put more work into the PR if needed.

**Comment by wbolster** [comment]
closed via #90
